ironfish/src/rpc/routes/wallet/createAccount.ts

```diff
@@ -18,7 +18,15 @@ import { AssertHasRpcContext } from '../rpcContext'
  * Hence, we're adding a new createAccount endpoint and will eventually sunset the create endpoint.
  */
 
-export type CreateAccountRequest = { name: string; default?: boolean }
+export type CreateAccountRequest = {
+  name: string
+  default?: boolean
+  createdAt?: {
+    hash: string
+    sequence: number
+  }
```

Comment:
we're in the process of deprecating `hash` as part of `createdAt`. the `importAccount` RPC only uses a number for `createdAt`. should we do the same here?

Reply:
So shouldn't make a difference if we just set it to an empty buffer? `Buffer.alloc(32, 0)`